studybuddies/src/app/app.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7155399)
```suggestion
  selector: "app-root",
```